index.html

```diff
@@ -4116,91 +4076,298 @@ <h3>Parsing content</h3>
+          Denotes an explicit permission given by the user.
+          The [[[PERMISSIONS]]] API is suggested to be used by
+          UAs for implementing NFC related permissions in order to minimize
+          the need for user prompting.
+        </p>
+        <p>
+          The
+          <a href="https://www.w3.org/TR/permissions/#dictdef-permissiondescriptor">
+          <dfn>Web NFC permission name</dfn></a> is
+          <a href="https://github.com/w3c/permissions/issues/47">defined</a> as
+          "`nfc`".
+        </p>
+      </section>
+      <section> <h4><dfn>Ask for forgiveness</dfn></h4>
+        <p>
+          Denotes an unobtrusive way to notify the user about an ongoing operation
+          with a possible action to cancel or undo the operation, and to
+          handle future occurences of the same operation by the same action.
+        </p>
+      </section>
+      <section> <h4><dfn>Prearranged trust relationship</dfn></h4>
+        <div>
+          A prearranged trust relationship means the UA has already
+          established a trust relationship for a certain operation using a
+          platform specific mechanism, so that an <a>expressed permission</a>
+          from the user is not any more needed. A few examples are given below.
+          <ul>
+            <li>
+              Web apps installed from a store, or web pages installed to home
+              screen (with [[appmanifest]]) MAY be considered trusted by the user
+              agent. In this case the trust concerns the web page and allowed
+              NFC functionality, but does not necessarily also concern the
+              data exchanged through NFC.
+            </li>
+            <li>
+              For trusting the integrity of the data exchanged via NFC, user
+              agents MAY use encrypted <a>NFC content</a> with key management
+              based on Public Key Infrastructure.
+              Key management is out of the scope of Web NFC.
+              <p class="note">
+                This is different from application level end to end encryption.
+                Key management is then in application domain.
+              </p>
+            </li>
+            <li>
+              User agents MAY consider using <a>NDEF signature</a> records for
+              <a>NFC content</a> integrity. When <a>NDEF record</a>s are signed
+              with a site certificate, then that is considered
+              <a>prearranged trust relationship</a> and browser origin policy
+              MAY be applied to relax the security policies in this document,
+              in the case when the user agent wants to minimize or eliminate
+              user prompts involved in obtaining permissions.
+              The signing tools and Public Key Infrastructure for key management
+              are out of scope of Web NFC.
+            </li>
+          </ul>
+        </div>
+      </section>
+    </section>
+
+    <section> <h3>Sign NDEF records</h3>
```

Comment:
Signed?

Reply:
Verb. Many mechanism titles use a verb. If we change, should be consistent.